[ActionSheet] Use in-framework header imports

Importing Bottom Sheet using framework-style imports is causing compilation breakages for an internal client.

@jverkoey So we have a slightly strange situation because we straddle the line of internal/external and have Swift modules that need to work on both. This means we use ModuleMaps to graft internal-style import patterns onto external MDC. You can see an example of that here: https://github.com/google/science-journal-ios/blob/master/ModuleMaps/MDC/ButtonBar/ButtonBar.modulemaps/module.modulemap

It's possible this has something to do with this issue, but honestly we have not seen this anywhere else, only in ActionSheet. That said, here's the error we get when we create an ActionSheet modulemap in the same style as above and then `import third_party_objective_c_material_components_ios_components_ActionSheet_ActionSheet`:

```ScienceJournal/google3/third_party/sciencejournal/ios/Pods/MaterialComponents/components/ActionSheet/src/MDCActionSheetController.h:17:2: note: in module 'MaterialComponents' imported from ScienceJournal/google3/third_party/sciencejournal/ios/Pods/MaterialComponents/components/ActionSheet/src/MDCActionSheetController.h:17:
#import <MaterialComponents/MaterialBottomSheet.h>
 ^
Library/Developer/Xcode/DerivedData/ScienceJournal-gvgfrzbykdhdoodvlvcbiepfuumm/Build/Products/Debug-iphonesimulator/MaterialComponents/MaterialComponents.framework/Headers/UIViewController+MaterialBottomSheet.h:22:12: warning: duplicate definition of category 'MaterialBottomSheet' on interface 'UIViewController'
@interface UIViewController (MaterialBottomSheet)
           ^
ScienceJournal/google3/third_party/sciencejournal/ios/Pods/MaterialComponents/components/BottomSheet/src/UIViewController+MaterialBottomSheet.h:22:12: note: previous definition is here
@interface UIViewController (MaterialBottomSheet)
           ^
<module-includes>:2:9: note: in file included from <module-includes>:2:
#import "../../../../Pods/MaterialComponents/components/ActionSheet/src/MaterialActionSheet.h"
        ^
ScienceJournal/google3/third_party/sciencejournal/ios/ModuleMaps/MDC/ActionSheet/ActionSheet.modulemaps/../../../../Pods/MaterialComponents/components/ActionSheet/src/MaterialActionSheet.h:15:9: note: in file included from ScienceJournal/google3/third_party/sciencejournal/ios/ModuleMaps/MDC/ActionSheet/ActionSheet.modulemaps/../../../../Pods/MaterialComponents/components/ActionSheet/src/MaterialActionSheet.h:15:
#import "MDCActionSheetController.h"
        ^
ScienceJournal/google3/third_party/sciencejournal/ios/Pods/MaterialComponents/components/ActionSheet/src/MDCActionSheetController.h:41:46: error: duplicate interface definition for class 'MDCActionSheetController'
__attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController : UIViewController
                                             ^
Library/Developer/Xcode/DerivedData/ScienceJournal-gvgfrzbykdhdoodvlvcbiepfuumm/Build/Products/Debug-iphonesimulator/MaterialComponents/MaterialComponents.framework/Headers/MDCActionSheetController.h:41:57: note: previous definition is here
__attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController : UIViewController```

I think the issue here is that pulling it in like a framework means the import is pulling in MaterialComponents/BottomSheet, but then we're also pulling it in via our module. I don't have a good answer for this off the top of my head.

--- a/components/ActionSheet/src/MDCActionSheetController.h
+++ b/components/ActionSheet/src/MDCActionSheetController.h
@@ -14,7 +14,7 @@
 
 #import <UIKit/UIKit.h>
 
-#import <MaterialComponents/MaterialBottomSheet.h>
+#import "MaterialBottomSheet.h"
 
 @class MDCActionSheetAction;
 

